ReSharper.FSharp/src/FSharp.Psi.Intentions/src/QuickFixes/GenerateMissingRecordFieldsFix.fs

```diff
@@ -121,3 +112,139 @@ type GenerateMissingRecordFieldsFix(recordExpr: IRecordExpr) =
+            let declaredField = declaredFields[fieldIndex]
+            let createdBinding = indexedBindings[fieldIndex]
+
+            if isNull createdBinding then
+                let binding = elementFactory.CreateRecordFieldBinding(declaredField, generateSingleLine)
+
+                let actualBinding =
+                    if fieldIndex = 0 then
+                        if isNull recordExpr.FieldBindingList then
+                            let bindingList = RecordFieldBindingListNavigator.GetByFieldBinding(binding)
+                            let actualList = ModificationUtil.AddChildAfter(recordExpr.LeftBrace, bindingList)
+                            actualList.FieldBindings.First()
+                        else
+                            let anchor = recordExpr.FieldBindingList.FieldBindings.First()
+                            ModificationUtil.AddChildBefore(anchor, binding)
+                    else
+                        let anchor: ITreeNode =
+                            let indexedBinding = indexedBindings[fieldIndex - 1]
+                            if generateSingleLine then
+                                indexedBinding
+                            else
+                                x.SkipComments indexedBinding
+
+                        let resultingNode =
+                            // Nodes after block comments are not automatically moved to the new line, fixing it
+                            if (not generateSingleLine) && anchor.GetTokenType() == FSharpTokenType.BLOCK_COMMENT then
+                                let newLineNode = NewLine(binding.GetLineEnding())
+                                let insertedNewLine = ModificationUtil.AddChildAfter(anchor, newLineNode)
+                                ModificationUtil.AddChildAfter(insertedNewLine, binding)
+                            else
+                                ModificationUtil.AddChildAfter(anchor, binding)
+
+                        resultingNode
+
+                indexedBindings[fieldIndex] <- actualBinding
+                generatedBindings.AddLast(actualBinding) |> ignore
+            else
+                if generateSingleLine && isNull createdBinding.Semicolon then
+                    addSemicolon createdBinding
+
+        generatedBindings
+
+    member private x.SkipComments (node: ITreeNode): ITreeNode =
+        let mutable current = node
+        let mutable found = false
+
+        while not found do
+            let next = current.NextSibling
+            if next == null then found <- true
+
+            else if next.GetTokenType() == FSharpTokenType.WHITESPACE then
+                let nextNext = next.NextSibling
+                if nextNext == null then found <- true
+
+                // Whitespace + Comment: set comment as current
+                else if nextNext.IsCommentToken() then
+                    current <- nextNext
+
+                else found <- true
+
+            else found <- true
+
+        current
```

Comment:
It looks like this can be replaced with `getLastMatchingNodeAfter isInlineSpaceOrComment node` on the callee side.